recipe/meta.yaml

```diff
@@ -1,33 +1,39 @@
 {% set name = "qstylizer" %}
-{% set version = "0.1.10" %}
+{% set version = "0.2.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/blambright/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
-  sha256: a5d561ef757d1751766674fba977e339cb4028b057a350969224403f286c4751
+  sha256: 95f7666929cc34efbbd9e0640a13f507b8ec35ec34a9604017cf96c24f7f9ea6
 
 build:
-  noarch: python
   number: 0
+  skip: true  # [s390x]
+  #Inflection is not available for s390x platform. https://anaconda.org/main/inflection
 
 requirements:
   host:
     - python >=3.6
     - pip
     - pbr
+    - setuptools
+    - wheel
   run:
     - python >=3.6
-    - tinycss >=0.4,<1.0
+    - tinycss2 >=1.1.0,<2.0
     - inflection >0.3.0,<1.0
 
-
 test:
   imports:
     - qstylizer
     - qstylizer.style
+  require:
```

Comment:
Must be `requires`, not `require`. That's why tests failed. But conda-build logs don't say it explicitly

Reply:
Thanks for noticing this!